source/chapters/appendix.rst

```diff
@@ -296,6 +296,12 @@ file(s) on the command line. These will be loaded at start-up.  Each file you
 specify will be loaded into the next virtual deck. For a list of supported file
 types, go to :ref:`configuration-import`.
 
+To launch Mixxx using the command line:
+For Windows OS, open the folder where Mixxx is installed and copy the file path. You can also obtain the file path by right clicking on the Mixxx icon and clicking 'Properties'. The path will usually look like ``C:\Program Files\Mixxx`` or ``C:\Program Files (x86)\Mixxx``. Change directory to this path in the command line by typing ``cd C:\Program Files\Mixxx``. Once the command prompt is at this location, type ``mixxx`` and press Enter.
```

Comment:
It's not necessary to `cd` to it, you can can also just type `C:\Program Files\Mixxx\mixxx --developer` directly.

Reply:
Please also add information about the other OSes. On Linux, you can just type `mixxx` (assuming the `mixxx` executable is in `$PATH`). @ronso0 explained in #272 how to run it on macOS. I think we should use a layout like here for the different OSes: https://manual.mixxx.org/2.2/en/chapters/getting_started.html#opening-mixxx

Reply:
Ohh yes! This looks better. Let me make the required changes.

Reply:
> It's not necessary to `cd` to it, you can can also just type `C:\Program Files\Mixxx\mixxx --developer` directly.

Hi @Holzhaus I tried this option and it did not seem to work for me. I tried different options but this is what i got

![image](https://user-images.githubusercontent.com/28389950/100883642-9d75af00-34c1-11eb-9257-87c9657bf58c.png)

I was only successful with `cd` then `mixxx`.
What do you think?

Reply:
I think you need quotes around the full path name so the spaces are not interpreted as separating command arguments:
```
"C:\Program Files\Mixxx\mixxx --developer"
```

Reply:
Also, the command specified [here](https://mixxx.org/manual/latest/en/chapters/appendix.html#developer-tools) for starting Mixxx in Developer mode from a custom resource directory with MIDI and HID logging enabled might also be OS specific.
Should I change this to state that the user can simply append `--controllerDebug --developer --resourcePath res` to the base command depending on their operating system?

Reply:
Hey @ronso0 I would love to get your thoughts on this as well.

Reply:
> I think you need quotes around the full path name so the spaces are not interpreted as separating command arguments:
> 
> ```
> "C:\Program Files\Mixxx\mixxx --developer"
> ```

That should be

    "C:\Program Files\Mixxx\mixxx" --developer

Reply:
> I think you need quotes around the full path name so the spaces are not interpreted as separating command arguments:
> 
> ```
> "C:\Program Files\Mixxx\mixxx --developer"
> ```

Hi @Be-ing, I have just tried that as well. It still didn't work.

![image](https://user-images.githubusercontent.com/28389950/100887317-e0398600-34c5-11eb-8152-191866e40c4b.png)

Reply:
The path for `cd` should not include the `mixxx` executable:
```
cd "C:\Program Files\Mixxx"
mixxx --developer
```

```
"C:\Program Files\Mixxx\mixxx" --developer
```

I think those would both work.

Reply:
Yes. They worked! Thank you.